packages/main/src/mainWindow.ts

```diff
@@ -26,6 +26,15 @@ async function createWindow() {
     }
   });
 
+  /**
+   * linsten ipcRenderer event from render process
+   */
+   ipcMain.on('renderMsg',(event,arg)=>{
+    console.log(arg);
+    event.sender.send('mainMsg','pong');
+  });
```

Comment:
Main should do something more useful. How about show Message Box? [`showMessageBox`](https://www.electronjs.org/docs/latest/api/dialog#dialogshowmessageboxbrowserwindow-options) with message from renderer and some two buttons. And send back to renderer dialog's response with index of the clicked button.